integration-test-groups/aws2/aws-secrets-manager/src/main/resources/application.properties

```diff
@@ -21,6 +21,6 @@ camel.component.aws-secrets-manager.access-key=${AWS_ACCESS_KEY}
 camel.component.aws-secrets-manager.secret-key=${AWS_SECRET_KEY}
 camel.component.aws-secrets-manager.region=${AWS_REGION:us-east-1}
 camel.component.aws-secrets-manager.useDefaultCredentialsProvider=${AWS_USE_DEFAULT_CREDENTIALS_PROVIDER}
-
+localstack.container.image=mirror.gcr.io/localstack/localstack:3.7.2
```

Comment:
Please remove this.

Reply:
Sorry, I missed that - it was for my idea debugging.

Reply:
removed